packages/devtools/lib/src/memory/memory.dart

```diff
@@ -96,13 +105,35 @@ class MemoryScreen extends Screen with SetStateMixin {
 
     final CoreElement screenDiv = div(c: 'custom-scrollbar')..layoutVertical();
 
+    html.window.onKeyDown.listen((html.KeyboardEvent e) {
```

Comment:
do you really want to listen on the window? 
As is, this would cause problems where pressing control-f would always trigger regardless of what view you are in and this event listener would never go away even if the Memory page was removed from the app.

Reply:
Probably not.  Added shortcutCallback to framework as a mechanism to handle shortcuts for any screen.